demo-samples/styles.css

```diff
@@ -0,0 +1,107 @@
+*, *::before, *::after {
+  box-sizing: inherit;
+}
+
+@at-root {
+  @-moz-viewport      { width: device-width; }
+  @-ms-viewport       { width: device-width; }
+  @-o-viewport        { width: device-width; }
+  @-webkit-viewport   { width: device-width; }
+  @viewport           { width: device-width; }
+}
+
+[tabindex="-1"]:focus {
+  outline: none !important;
+}
+
+html, body {
+  margin: 0;
+  padding: 0;
+  font-size: 16px;
+  box-sizing: border-box;
+}
+
+html {
+  -webkit-tap-highlight-color: rgba(0,0,0,0);
+}
+
+body {
+  font-family: 'OpenSans', sans-serif;
+  font-family: $font-family-base;
+  line-height: 1.4;
+  color: #ececec;
+  background-color: #333;
+}
+
+.map-container {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  width: 100%;
+}
+
+#map {
+  width: 100%;
+  height: 100%;
+}
+
+.legend {
+  position: fixed;
+  right: 2rem;
+  bottom: 3rem;
+  display: flex;
+  width: 180px;
+  height: 100px;
+  padding-bottom: 3rem;
+  background-color: rgba(70,70,70,.8);
+}
+
+.legend ul {
+  list-style: none;
+  margin: 0;
+  padding; 0;
+}
+.legend li {
+  position: absolute;
+  left: 65%;
+  bottom: 15px;
+  transform: translateX(-50%);
+}
+li .circle {
+  display: block;
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    height: 100%;
+    border-radius: 50%;
+    opacity: .8;
+}
+li .label {
+  position: absolute;
+  top: 0;
+  right: 50%;
+  width: 100px;
+  display: flex;
+  align-items: flex-start;
+  justify-content: flex-start;
+}
+li .label:after {
+  content: '';
+  display: block;
+  flex: 1;
+  height: 0;
+  border-top: 1px dotted #ccc;
+}
+li .label-val {
+  margin-right: 5px;
+  margin-top: -5px;
+  color: #ccc;
+  text-align: right;
+  width: 30px;
+  font-size: 10px;
+  line-height: 1;
+}
+
+.mapboxgl-popup {
```

Comment:
I think the `body.color` style may be affecting the popup font color:
![image](https://user-images.githubusercontent.com/1508618/76559084-77f4d100-646c-11ea-985e-d8042ed1f4d3.png)
If we add a color property here, will that override it?

Reply:
Nice catch - will fix!